domains/djstudio.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "DJOPRO-STUDIO",
+    "email": "tuto11117@hotmail.com"
+  },
+  "record": {
+    "URL": "https://google.com"
```

Comment:
Please use your website link.
is-a.dev is a free service for developers to have a custom subdomain for **their website**